addons/purchase/models/product.py

```diff
@@ -75,9 +74,8 @@ def action_view_po(self):
         action = self.env.ref('purchase.action_purchase_order_report_all').read()[0]
         action['domain'] = ['&', ('state', 'in', ['purchase', 'done']), ('product_id', 'in', self.ids)]
         action['context'] = {
-            'search_default_last_year_purchase': 1,
-            'search_default_status': 1, 'search_default_order_month': 1,
-            'graph_measure': 'qty_ordered'
+            'graph_measure': 'qty_ordered',
+            'search_default_later_than_a_year_ago': 1
```

Comment:
As we are at it, I would prefer to use True instead of 1 (as there is no order and as it's about enabling a search - so it's more a bool than a int). The result is the same but I found 'True' clearer in this case.

Reply:
Done!